console-consumer-primitive-keys-values/README.md

```diff
@@ -0,0 +1,61 @@
+# Consuming non-string primitive keys and values 
```

Comment:
I forget what we decided about directory structure for tutorials like this that aren't ksql / kstreams / flinksql. did we say we'd have a `kafka` directory? or leave as is with a README at the toplevel?

Reply:
I don't remember 😞

Reply:
yeah this seems better... @justin-paresky would this layout work ok for rendering on CD? i.e. for a tutorial that doesn't have sub-flavors (like `ksql` and `kstreams`), just have the README at the top level